apps/web/src/hooks/v3/useToggleNotifications.ts

```diff
@@ -32,5 +36,5 @@ export const useTogglenotifications = () => {
     }
   }, [setAllowNotifications, toast])
 
-  return { allowNotifications, handleEnableNotifications, handleDiableNotifications }
+  return { allowNotifications, featureEnabled, handleEnableNotifications, handleDiableNotifications }
```

Comment:
What about we simplify like this?
```suggestion
  return { enabled, toggle }
```

Reply:
enabled is true if we meet any of the following condition

1. The global settings is enabled
2. The experimental feature is enabled && global settings is uninitialized 

Global settings should have two states:
- Uninitialized, which means the value should be `undefined`
- Initialized, which means either `true` or `false`

So that we can make sure
1. The feature will stay on if the user turned it on manually, no matter he's within the ab testing group or not
2. The feature will stay off if the user turn it off manually, no matter he's within the ab testing group or not
3. User who doesn't toggle the global settings manually will follow the experimental feature flag, which is controlled by our ab testing strategy